app/models/application_reference.rb

```diff
@@ -46,4 +46,10 @@ def self.find_by_unhashed_token(unhashed_token)
     hashed_token = Devise.token_generator.digest(ApplicationReference, :hashed_sign_in_token, unhashed_token)
     find_by(hashed_sign_in_token: hashed_token)
   end
+
+  # TODO: Revisit this to factor out the time limit to a constant and use a
+  # dedicated `request_sent_at` as the start point.
+  def response_overdue?
+    feedback_requested? && created_at < 10.business_days.ago
```

Comment:
Is this something we should include in the `TimeLimitConfig` rules?

Reply:
Yes I think this is what Tijmen did in another open PR. I will change it here to be consistent - though I think we might need to revisit the logic.